Installation fail on Yunohost 11.0.10.2

### Describe the bug

The installation fails while installing the psycopg2cffi from the python requirements.txt, during the `Preparing metadata` step, with the error message : 
```
             c/_cffi_backend.c:15:10: fatal error: ffi.h: No such file or directory
                15 | #include <ffi.h>
                   |          ^~~~~~~
             compilation terminated.
             error: command '/usr/bin/arm-linux-gnueabihf-gcc' failed with exit code 1
             [end of output]
 
         note: This error originates from a subprocess, and is likely not a problem with pip.
         ERROR: Failed building wheel for cffi
       ERROR: Failed to build one or more wheels
```
### Context

- Hardware: Other ARM board, A20-OLinuXino-Lime2
- YunoHost version: 11.0.10.2
- I have access to my server: everything
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: Yunohost installed over a fresh armbian
- Using, or trying to install package version/branch: 2.11.20220622~ynh1

### Steps to reproduce

1. Go to Webadmin  > Applications > Catalogue 
2. Look for mygpo (gPodder.net) 
3. Click Install
4. Label "My gPodder", select a domain not used by another app, not visible to anonymous users, select myself as app admin
5. Click Install

### Expected behavior

The app installs itself

### Logs

https://paste.yunohost.org/raw/wocoxugafe

--- a/scripts/_common.sh
+++ b/scripts/_common.sh
@@ -5,7 +5,7 @@
 #=================================================
 
 # dependencies used by the app
-pkg_dependencies="acl python3 python3-dev python3-pip python3-venv postgresql postgresql-contrib libpq-dev redis-server libldap2-dev libsasl2-dev"
+pkg_dependencies="acl libffi-dev libjpeg-dev libwebp-dev python3 python3-dev python3-pip python3-venv postgresql postgresql-contrib libpq-dev redis-server libldap2-dev libsasl2-dev"
 
 #=================================================
 # PERSONAL HELPERS

